nameSpace.js

```diff
@@ -0,0 +1,14 @@
+function provide(newClass, constructor) {
```

Comment:
Check my example form the comment above, the first argument is string type. The idea is that I can define a namespace + Class as a string and with it export the class. Examples:
```
provide("djani.kung.fu.Pera", SomeClass);
provide("zile.car.karatea", OtherClass);
```

Then when I want to use it somewhere in the code later on I can require it:
```
var OtherClass = require("zile.car.karatea");
var zileInstance = new OtherClass();
zileInstance.doSomethingForExample();
```